.docker/README.md

```diff
@@ -0,0 +1,9 @@
+# ROS2 Control Docker Containers
+
+Meant to provide basic docker containers for CI or development purposes.
+
+## Source folder and tag
+Downloads and builds ros2_controls into a workspace for development use exactly as is found [here](https://ros-controls.github.io/control.ros.org/getting_started.html#compiling).
```

Comment:
Can we please add a few copy-paste-able commands here for the general public to use?

something along the lines of
```
docker build ... --tag ros2_control_source
docker run --it ros2_control_source /bin/bash
```

Reply:
My recent push added in some example commands for both source and release tags